crfs.go

```diff
@@ -52,6 +52,21 @@ const debug = false
 
 var (
 	fuseDebug = flag.Bool("fuse_debug", false, "enable verbose FUSE debugging")
+
+	// A "whiteout" file is an empty file with a special filename that signifies a
```

Comment:
don't mix non-flags in the var ( ... ) flag block.

Also, these could be constants, right?  Use `const ( .... )`

Reply:
Also, each doc comment should start with a subject that is the symbol itself, per https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

  // whiteoutPrefix is ...